### ISSUE TITLE
[Windows] crashReporter not sending guid default property

### ISSUE THREAD

**Issue by popod** [electron/electron#15777]
* Output of `node_modules/.bin/electron --version`: 3.0.10
* Operating System (Platform and Version): Windows 10

**Expected Behavior**
The crash data sent to the server should contain all the defaults proportion listed on the [doc](https://electronjs.org/docs/api/crash-reporter#crash-report-payload).

**Actual behavior**
The crash data `guid` property is missing on crash data from Windows.

**To Reproduce**
No special config, just active crash reports on electron-quick-start as explain in the [doc](https://electronjs.org/docs/api/crash-reporter#crashreporter).

``` js
// both in main and renderer
crashReporter.start({
  companyName: 'someName',
  submitURL: 'https://www.somedomaine.com/crash-report',
  extra: {
    username: 'theUserName'
  }
})
```

**Comment by aabuhijleh** [comment]
It's also reproduced in Electron v4.0.4

**Comment by popod** [comment]
Hi @nitsakh I've seen that you do some improvements in #17978. Have you some skills to helping resolve this issue too ? Thanks in advance :)

**Comment by flashyu** [comment]
I also encountered the same problem

**Comment by popod** [comment]
It looks like this bug is fixed with Electron version 6.0.11. Could anyone else confirm that too?
